src/index.js

```diff
@@ -78,6 +86,10 @@ class SubstrateApp {
     return chunks
   }
 
+  static hd_key_derivation(mnemonic, account, change, addressIndex) {
```

Comment:
how is that we don't use the arguments here?

Reply:
ah this can actually be removed. I first thought to make it part of the app but better standalone (The app requires the "transport" to initialize, don't think we want that)